tools/standalone-binary-distro/make_distro_builds.sh

```diff
@@ -28,17 +28,21 @@ prepare() {
    echo "BUILD_LIBRARY_FOR_DISTRIBUTION = YES" >> $xcconfig
    # carthage may try to build schemes in dependency-manager-tests folder, so we hide it temporarily
    mv dependency-manager-tests/ .dependency-manager-tests/ 
+   # hide '*.local.xcconfigs' so they are not applied to produced artifacts (they are renamed to '*_bak'):
+   find . -type f -name '*.local.xcconfig' -exec mv '{}' '{}_bak' ';'
```

Comment:
💡 This change should be originally done to the `make_distro_builds.sh` to not apply local & git-ignored configuration to produced XCFrameworks. I'm adding it in this PR, as now it's even more necessary.

IMO it's good for now, but in general I see this patching approach very inconvenient - we need to patch repository before exporting XCFrameworks and then revert patches again to reset it to the original state. IMO it would be way easier and safer to export XCFrameworks from new clone of the repository, instead of current working directory. It's easy to imagine that some setup might be overlooked (like these `*.local.xcconfig`) leading to exporting bad artifacts.